packages/ibm-products/src/components/ConditionBuilder/ConditionBuilderContext/translationObject.js

```diff
@@ -1,40 +1,38 @@
 export const translationsObject = {
-  en: {
-    if: 'if',
-    excl_if: 'excl.if',
-    and: 'and',
-    or: 'or',
-    is: 'is',
-    greater: 'is greater than',
-    greater_equal: 'is greater than or equal to',
-    lower: 'is lower than',
-    lower_equal: 'is lower than or equal to',
-    starts_with: 'starts with',
-    ends_with: 'ends with',
-    contains: 'contains',
-    one_of: 'is one of',
-    before: 'is before',
-    after: 'is after',
-    between: 'is between',
-    add_condition: 'Add condition',
-    add_condition_group: 'Add condition group',
-    add_subgroup: 'Add subgroup',
-    condition: 'Condition',
-    property: 'Property',
-    operator: 'Operator',
-    value: 'Value',
-    connector: 'Connector',
-    condition_row: 'Condition row',
-    remove_condition: 'Remove condition',
-    add_condition_row: 'Add condition row',
-    start: 'Start',
-    end: 'End',
-    clear_search: 'Clear search input',
-    actions: 'Actions',
-    then: 'then',
-    remove_action: 'Remove Action',
-    add_action: 'Add action',
-    invalid_text: 'Incomplete',
-    text_invalid_number: 'Invalid number, must be 0 or greater',
-  },
+  if_text: 'if',
```

Comment:
can you switch the keys to use camel case? `ifText`

Reply:
Modified accordingly. But I could see in carbon , translate message keys used with hypens or underscore.